src/hooks/useAuthForm.tsx

```diff
@@ -18,17 +24,71 @@ export interface CommonFieldValues extends FieldValues {
   passwordConfirmation: string;
 }
 const useAuthForm = () => {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    const storedUser = localStorage.getItem('user');
+    if (storedUser !== null) {
+      dispatch(setUser(JSON.parse(storedUser)));
+    }
+  }, []);
+  const navigate = useNavigate();
   const {
     handleSubmit,
     register,
     formState: { errors },
   } = useForm<CommonFieldValues>();
 
   const onSignInSubmit: SubmitHandler<ISignIn> = (data: ISignIn) => {
-    console.log('Form data:', data);
+    axios
+      .post('/login', data)
+      .then(({ data }) => {
+        dispatch(
+          setUser({
+            token: data.accessToken,
+            ...data.user,
+          }),
+        );
+        localStorage.setItem(
```

Comment:
As a option read about redux-persist, I do not insist but this library can help you to store your data in LS

Reply:
Not changed at the moment but noted down